blog/2024-02-26-Get-Started-With-Uptane/index.md

```diff
@@ -0,0 +1,112 @@
+
+But what if you are coming at it from the opposite direction? You don't already have a big complex system to secure, and you just want to set up a software repository and deliver software updates to embedded devices, using best practices from the start? In short, now that you know what Uptane is and what Uptane isn't, how can you actually use it?
+
+## Ways to use Uptane Today
+
+So, for a full Uptane system, you need:
+
+* An image repository, with some APIs/interfaces for doing common things
+* A director repository, with some way to create software updates and assign them to devices
+* A client on your device that can talk to your server, and install updates
+
+There are actually pretty easy options for all of these things, including both free-as-in-beer and free-as-in-speech resources.
+
+### Servers/repositories
+
+* [OTA Community Edition](https://github.com/uptane/ota-community-edition/)
+
+OTA Community Edition is the open-source, community-maintained Uptane implementation. It implements an image repository, director repository, and a device registry to keep track of the manifests (and other information) the devices send. It has HTTP REST APIs for all of the basic necessities for Uptane, but also for the practical bits that you need, like actually assigning updates to devices, uploading software, generating keys, and so on.
+
+The OTA Community edition repository contains instructions for how to spin up the services on a local network; to run an adequate production instance you'll need to fill in some blanks. It doesn't currently have a well-maintained web UI, but it's still possible to use the UI that was previously developed and open-sourced by HERE Technologies, once you make a few modifications to stub out API calls to their proprietary services. (P.S.: If you're interested in contributing to Uptane, developing a community-maintained web UI for OTA Community Edition is one of the [project proposals we're interested in](https://wiki.linuxfoundation.org/gsoc/2024-gsoc-uptane) for [Google Summer of Code 2024](https://summerofcode.withgoogle.com/)!)
+
+* [app.uptanedemo.org](http://app.uptanedemo.org)
+
+I've spun up an instance of OTA Community Edition to help people get their feet wet, for free. It's open to the public as a demo instance, with all of its private signing keys published publicly (so, buyer beware: if you connect a device to this instance, anyone who wanted to could install malicious software). I wipe and reset it periodically. Some documentation on how to use it is coming soon, but for now, just pop into discord and ask if you'd like to connect a device to it.
```

Comment:
```suggestion
I've spun up an instance of the OTA Community Edition to help people get their feet wet, for free. It's open to the public as a demo instance, with all of its private signing keys published publicly (so, buyer beware: if you connect a device to this instance, anyone who wanted to could install malicious software). I wipe and reset it periodically. Some documentation on how to use it is coming soon, but for now, just pop into Discord and ask if you'd like to connect a device to it.
```

Reply:
I don't think the definite article is correct here, because "OTA Community Edition" is the name of the software project.

```suggestion
I've spun up an instance of OTA Community Edition to help people get their feet wet, for free. It's open to the public as a demo instance, with all of its private signing keys published publicly (so, buyer beware: if you connect a device to this instance, anyone who wanted to could install malicious software). I wipe and reset it periodically. Some documentation on how to use it is coming soon, but for now, just pop into Discord and ask if you'd like to connect a device to it.
```